script/Arcueid/Day 5/05_00_ARC05_1.txt

```diff
@@ -299,12 +299,13 @@ Not a single victim has been discovered―――?
 [sha:9945f275c190c472f276e6b4aecace1a00ee64ce]{
 -- Page 25, Offset 8650.
 -- なぜそこまで解っていてはっきりと口にしない。
-How can you say that when you have so much information?
+// im not sure this is right, but the newscaster shouldn't know what really happened right?
+How can you say that when we already know?
 }
 [sha:c631ecbf4517c34f730ddf33ccbea27070809a1b]{
 -- Page 25, Offset 8651.
 -- ホテルに泊まっていた人間はみんな、あのワケのわからない炎にまかれて、跡形もなく燃え尽きてしまった事を。
-The information that all the people that stayed at the hotel were engulfed in those unfathomable flames and burned without a trace.
+That everyone that stayed in the hotel was engulfed in those unfathomable flames, burnt to a cinder without leaving any trace.
```

Comment:
"That everyone who stayed"

"burned to a crisp" is the expression but "reduced to [cinder|ash] without leaving a single trace" might be a little nicer.